src/test/java/uk/gov/crowncommercial/dts/scale/cat/util/TestUtils.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.crowncommercial.dts.scale.cat.util;
+
+import java.util.Arrays;
+import java.util.List;
+
+import uk.gov.crowncommercial.dts.scale.cat.model.agreements.ProjectEventType;
+import uk.gov.crowncommercial.dts.scale.cat.model.generated.DefineEventType;
+import uk.gov.crowncommercial.dts.scale.cat.model.generated.EventType;
+
+public class TestUtils {
+
+  public static final String EXPRESSION_OF_INTEREST = "Expression of Interest";
+  public static final String REQUEST_FOR_INFORMATION = "Request for Information";
+  public static final String CAPABILITY_ASSESSMENT = "Capability Assessment";
+  public static final String DIRECT_AWARD = "Direct Award";
+  public static final String FURTHER_COMPETITION = "Further Competition";
+
+  public static EventType getEventType(DefineEventType defineEventType, String description,
+      boolean preMarketActivity) {
+    var eoiEventType = new EventType();
+    eoiEventType.setType(defineEventType);
+    eoiEventType.setDescription(description);
+    eoiEventType.setPreMarketActivity(preMarketActivity);
+    return eoiEventType;
+  }
+
+
```

Comment:
Formatting issues - double line spacing here, and no line spacing between the methods below. Ideally would have it auto format on save (do in Eclipse via save actions, not sure about IntelliJ), and need to get the formatters aligned really